components/breadcrumbs/README.md

```diff
@@ -9,32 +9,47 @@ The `d2l-breadcrumbs` element can be used to help users understand where they ar
 <!-- docs: demo display:block -->
 ```html
 <script type="module">
-  import '@brightspace-ui/core/components/breadcrumbs/breadcrumb-current-page.js';
   import '@brightspace-ui/core/components/breadcrumbs/breadcrumbs.js';
 </script>
 <d2l-breadcrumbs>
   <d2l-breadcrumb text="Item 1" href="page1.html"></d2l-breadcrumb>
   <d2l-breadcrumb text="Item 2" href="page2.html"></d2l-breadcrumb>
-  <d2l-breadcrumb-current-page text="Current Page"></d2l-breadcrumb-current-page>
+  <d2l-breadcrumb text="Item 3" href="page3.html"></d2l-breadcrumb>
```

Comment:
If you click these does it just link you to a "Page not found"?

Reply:
Yeah, the iframe goes to a "Playground file not found". We could maybe just have the click do nothing?

Reply:
I've been leaving the href as `#` in other examples and it seems to give the desired effect